tuned/profiles/functions/function_package2cpus.py

```diff
@@ -0,0 +1,59 @@
+import os
+import fnmatch
+
+from . import base
+from tuned.utils.commands import commands
+
+cmd = commands()
+
+SYSFS_DIR = "/sys/devices/system/cpu/"
+
+class package2cpus(base.Function):
+	"""
+	Provides cpu device list for a package (socket)
+	"""
+
+	def __init__(self):
+		# 1 argument
```

Comment:
Could it make sense to have 1 to infinite arguments? E.g. If I want to get devices for package 1 and 2:
```ini
devices=${f:package2cpus:1:2}
```
If such queries make sense, the constrain could be relaxed and the arguments processed in the loop.